README.rst

```diff
@@ -1,17 +1,20 @@
 readthedocs-sphinx-search
 =========================
 
-|docs| |license| |build-status|
+|pypi| |docs| |license| |build-status|
 
-Sphinx extension to enable `search as you type` feature for docs hosted by Read the Docs.
+`Sphinx`_ extension to enable *search as you type* within a full-page search UI for docs hosted on `Read the Docs`_.
+Try it at https://readthedocs-sphinx-search.readthedocs.io/en/latest/?rtd_search=testing.
 
+.. _Sphinx: https://www.sphinx-doc.org/
+.. _Read the Docs: https://readthedocs.org/
 
 Installation
 ------------
 
 ::
 
-   pip install git+https://github.com/readthedocs/readthedocs-sphinx-search@master
+   pip install readthedocs-sphinx-search
```

Comment:
Does this work when there's only an `rc` release?

Reply:
yeah

```
$ pip install -i https://test.pypi.org/simple/ readthedocs-sphinx-search
Looking in indexes: https://test.pypi.org/simple/
Collecting readthedocs-sphinx-search
  Downloading https://test-files.pythonhosted.org/packages/57/50/849e38d298647d15eeb3e03438a58fc036c0f3b0bec14437803ee1
f81e95/readthedocs_sphinx_search-0.1.0rc2-py3-none-any.whl (25 kB)
Installing collected packages: readthedocs-sphinx-search
Successfully installed readthedocs-sphinx-search-0.1.0rc2
```